src/main/java/uk/gov/companieshouse/web/pps/controller/BaseController.java

```diff
@@ -18,14 +18,33 @@ public abstract class BaseController {
 
     protected static final String ERROR_VIEW = "error";
 
+    public static final String BACK_LINK_ATTR = "backLink";
+    public static final String USER_BAR_ATTR = "userBar";
+    public static final String USER_SIGNOUT_URL_ATTR = "userSignoutUrl";
+    public static final String HIDE_YOUR_DETAILS_ATTR = "hideYourDetails";
+    public static final String HIDE_RECENT_FILINGS_ATTR = "hideRecentFilings";
+    public static final String PHASE_BANNER_ATTR = "phaseBanner";
+    public static final String PHASE_BANNER_LINK_ATTR =  "phaseBannerLink";
+
     protected BaseController() {
     }
 
     @ModelAttribute("templateName")
     protected abstract String getTemplateName();
 
     protected void addBackPageAttributeToModel(Model model, String... pathVars) {
+        model.addAttribute(BACK_LINK_ATTR, navigatorService.getPreviousControllerPath(this.getClass(), pathVars));
+    }
+
+    protected void addUserModel(Model model) {
+        model.addAttribute(USER_BAR_ATTR, "1");
```

Comment:
According to the code from common-web module, th:if="${userBar}"
If there is any value set, it will be shown, not show if it is not set
I see some other using "1", so just follow that.

Reply:
That's fine if there's other examples, if it's a case of passing any value then we can keep as is, might be worth adding comment above this line to explain.

Reply:
Comment is added into the controller